python/ffsim/tenpy/util.py

```diff
@@ -0,0 +1,56 @@
+# (C) Copyright IBM 2024.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+from __future__ import annotations
+
+from tenpy.networks.mps import MPS
+from tenpy.networks.site import SpinHalfFermionSite
+
+
+def product_state_as_mps(bitstring: tuple[str, str]) -> MPS:
+    r"""Return the product state as an MPS.
+
+    Args:
+        bitstring: The bitstring in the form `(string_a, string_b)`.
+
+    Returns:
+        The product state as an MPS.
+    """
+
+    # extract norb
+    assert len(bitstring[0]) == len(bitstring[1])
+    norb = len(bitstring[0])
```

Comment:
```suggestion
    string_a, string_b = bitstring
    assert len(string_a) = len(string_b)
    norb = len(string_a)
```
Unpacking the strings will improve readability.